build-info-extractor-npm/src/main/java/org/jfrog/build/extractor/npm/extractor/NpmBuildInfoExtractor.java

```diff
@@ -159,6 +162,20 @@ private void createTempNpmrc(Path workingDir, List<String> installationArgs) thr
         }
     }
 
+    /**
+     * Boolean argument can be provided in one of the following ways:
+     *  1. --arg - which infers true
+     *  2. --arg=value (true/false)
+     *  3. --arg value (true/false)
+     */
+    private boolean isJsonOutputRequired(List <String> installationArgs) {
+        int jsonIndex = installationArgs.indexOf("--json");
+        if (jsonIndex > -1 ) {
+            return jsonIndex == installationArgs.size() - 1 || !installationArgs.get(jsonIndex).equals("false");
```

Comment:
I wonder if this code would return false if we have`--json=false`. 🤔 
I think that this function deserved a unit test.